test/e2e/pods.go

```diff
@@ -155,6 +156,42 @@ var _ = Describe("Pods", func() {
 			},
 		})
 	})
+	It("should be schedule with cpu and memory limits", func() {
+		podClient := c.Pods(api.NamespaceDefault)
+
+		By("creating the pod")
+		name := "pod-update-" + string(util.NewUUID())
```

Comment:
nit: Don't need the UUID when it's running in its own namespace as per previous comment.  Defer to sep PR.